### PR TITLE
Feature/suit 9003 Add tvOS element properties to JS API

### PR DESCRIPTION
**Pull request by JeKachan**

### CODE REVIEW

File: lib/constants/state.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const state = {
```
Comment (andriichumak) [inline lib/constants/state.js]: It's not clear from the name what this variable is about. Please, make naming more specific.

File: index.d.ts [hunk 2]
```diff
@@ -67,6 +71,10 @@ declare namespace suitest {
 		VRC: VrcConstants;
 		VIDEO_STATE: VideoState;
 		VISIBILITY_STATE: VisibilityState;
+		CONTENT_MODE: ContentMode;
+		STATE: State;
```
Comment (andriichumak) [inline index.d.ts]: Naming should be more specific for state.

File: lib/constants/state.js (hunk 1 above)
Comment (JeKachan) [reply lib/constants/state.js]: tvOSElementState?

Comment (andriichumak) [reply lib/constants/state.js]: How about `elementState`?

Comment (andriichumak) [reply lib/constants/state.js]: Also, we should keep it consistent with other constants. E.g. there is `ELEMENT_PROPS` (not `elementProps`). Please update new constants to fit that naming convention

**Pull request merged**